src/basic/raw-clone.h

```diff
@@ -11,6 +11,7 @@
 
 #include "log.h"
 #include "macro.h"
+#include "process-util.h"
```

Comment:
why is this added here rather than to the test?

Reply:
Because `reset_cached_pid()` is called below. This essentially fixes a pre-existing error.

Reply:
I added a comment to the commit message.